tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
can we use `"strict": true` (at least with the exception of `"noImplicitAny": false`) or will that be part of the next iteration?

Reply:
"strict": true,
    702 errors, maybe in next iteration